include/ord/Design.h

```diff
@@ -73,6 +73,10 @@ class Design
 
   int micronToDBU(double coord);
 
+  int evalTclString(const std::string& cmd);
```

Comment:
Please add a comment that this is a temporary backdoor and not a permanent feature.

Reply:
Will do. (CI in progress)